app/models/user.rb

```diff
@@ -13,7 +13,7 @@ class User < ActiveRecord::Base
     :recoverable, :rememberable, :trackable, :validatable,
     :omniauthable, omniauth_providers: [:facebook, :gplus]
 
-  has_many :classifications
+  has_many :classifications, dependent: :restrict_with_exception
```

Comment:
Just confirming, this restriction will effectively never allow a contributing user to be deleted...correct?